### PR TITLE
pharnb

### PR DESCRIPTION
**Pull request by pharnb**

### CODE REVIEW

**Comment by pharnb** [comment]
@rtrp/admins

File: users/pharnb/index.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ Ben Pharn
 
 I am a first year Master's student here at Oregon State University, having completed my undergraduate degree here as well.  I was born and raised in Southern California, where the majority of my family still resides.
 
-[Email] <a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
-[Resume] (./files/resume.pdf)
+[Email]<a href="mailto:pharnb@oregonstate.edu" target="top"> pharnb@oregonstate.edu </a> (pharnb@oregonstate.edu)
+/n [Resume](./files/resume.pdf)
```
Comment (dougwoods3) [inline users/pharnb/index.md:14]: This won't put it on a new line. I just tested using a double space at the end of your first line:

first line<space><space>
second line  
third line

and it works.

**Comment by dougwoods3** [comment]
In general, you can preview your edits on your browser by going to your index.md file.

File: users/pharnb/index.md (hunk 1 above)
Comment (dougwoods3) [reply users/pharnb/index.md:14]: oops, my example was supposed to say:

first line< space >< space >
second line < space >< space >
third line

An alternative is hitting enter twice and creating a new paragraph.

Comment (pharnb) [reply users/pharnb/index.md:14]: I put in the double-space.  I was looking to get it on the line immediately following, rather than after a gap.

**Comment by albeanth** [comment]
@pharnb  @dougwoods3 , is this ready to go?

**Comment by pharnb** [comment]
Yep.

**Comment by dougwoods3** [comment]
@pharnb You still have the \n instead of the double space.

**Comment by pharnb** [comment]
Really? Crap. I was sure I fixed that.

**Pull request merged**